### ISSUE TITLE
Default enable eod-mask-loss

### ISSUE THREAD

**Issue by luofenggu** [THUDM/CodeGeeX#94]
Should the `eod-mask-loss` be enabled by default, given that EOD has significant interference with the loss calculation for the "short answer" task?

**Comment by Stanislas0** [comment]
> Should the `eod-mask-loss` be enabled by default, given that EOD has significant interference with the loss calculation for the "short answer" task?

Yes! `eod-mask-loss` is recommended if most of training data are significantly shorter than 2048 tokens.

**Comment by luofenggu** [comment]
Thanks. I think CodeGeeX should initialize `eod-mask-loss =True` unless user manually disable it. 
It tooks me two days to discover that the reason for my problem was because I forgot to set the eod-mask-loss. T_T
